tests/components/lovelace/test_resources.py

```diff
@@ -137,38 +185,64 @@ async def test_storage_resources_import(
     # Update a resource
     first_item = response["result"][0]
 
-    await client.send_json(
+    await client.send_json_auto_id(
         {
-            "id": 8,
             "type": "lovelace/resources/update",
             "resource_id": first_item["id"],
             "res_type": "css",
             "url": "/local/updated.css",
         }
     )
+    response = await client.receive_json()
+    assert response["id"] == event_id
+    assert response["event"] == [
+        {
+            "change_type": "updated",
+            "item": {
+                "id": first_item["id"],
+                "type": "css",
+                "url": "/local/updated.css",
+            },
+            "resource_id": first_item["id"],
+        }
+    ]
+
     response = await client.receive_json()
     assert response["success"]
 
-    await client.send_json({"id": 9, "type": list_cmd})
+    await client.send_json_auto_id({"type": list_cmd})
     response = await client.receive_json()
     assert response["success"]
 
     first_item = response["result"][0]
     assert first_item["type"] == "css"
     assert first_item["url"] == "/local/updated.css"
 
-    # Delete resources
-    await client.send_json(
+    # Delete a resource
+    await client.send_json_auto_id(
         {
-            "id": 10,
             "type": "lovelace/resources/delete",
             "resource_id": first_item["id"],
         }
     )
+    response = await client.receive_json()
+    assert response["id"] == event_id
+    assert response["event"] == [
+        {
+            "change_type": "removed",
+            "item": {
+                "id": first_item["id"],
+                "type": "css",
+                "url": "/local/updated.css",
+            },
+            "resource_id": first_item["id"],
+        }
+    ]
+
     response = await client.receive_json()
     assert response["success"]
 
-    await client.send_json({"id": 11, "type": list_cmd})
+    await client.send_json_auto_id({"type": list_cmd})
```

Comment:
Could we extend the test so that when we unsubscribe, we don't get any more events?

Reply:
I don't know a good pattern for testing that, it's a problem in pretty much all tests related to WS subscriptions

Reply:
Can we not only unsubscribe from it, make a change, and then check that no WS response was received?

Reply:
Of course, that'd be great, but I don't know a pattern for that which doesn't suck.

Reply:
As we both don't know a nice way to test it and as the unsubscribe  mechanism is not new in this PR, we can skip the test